site/app/views/NavigationView.php

```diff
@@ -215,7 +214,8 @@ public function showGradeables($sections_to_list, $graded_gradeables, array $sub
             "display_room_seating" => $display_room_seating,
             "seating_only_for_instructor" => $this->core->getConfig()->isSeatingOnlyForInstructor(),
             "gradeable_title" => $gradeable_title,
-            "seating_config" => $seating_config
+            "seating_config" => $seating_config,
+            "DATE_TIME_FORMAT" => DateUtils::DATE_TIME_FORMAT
```

Comment:
Can we use https://twig.symfony.com/doc/3.x/functions/constant.html instead?

Reply:
I changed all these in my next PR so don't worry about this, it will be changed.